gitter/nlp/splitter.py

```diff
@@ -0,0 +1,23 @@
+import nltk
+
+
+class Splitter(object):
+
+    def __init__(self):
+        nltk.download('punkt')
```

Comment:
@jayvdb, Is there any other way to download this and `nltk.download('averaged_perceptron_tagger')` ? As of now its running every time we analyze a message.

Reply:
yes, do it in the module scope, or better create a single django command to download them all